src/OrbitGl/TimeGraph.cpp

```diff
@@ -96,6 +96,14 @@ TimeGraph::TimeGraph(AccessibleInterfaceProvider* parent, OrbitApp* app,
   vertical_slider_->SetOrthogonalSliderPixelHeight(horizontal_slider_->GetPixelHeight());
   horizontal_slider_->SetOrthogonalSliderPixelHeight(vertical_slider_->GetPixelHeight());
 
+  plus_button_ = std::make_shared<orbit_gl::PlusButton>(/*parent=*/this, viewport, &layout_);
+  plus_button_->SetMouseReleaseCallback([&](orbit_gl::Button* /*button*/) { ZoomTime(1, 0.5); });
+  plus_button_->SetLabel("Plus Button");
```

Comment:
There's a PlusButton class, should we set the label there instead?

Reply:
(Personal preference) I would like to set labels for MinusButton and PlusButton in their own constructors.

You can also set the mouse release callback in the constructor.

Reply:
Yes, it's true. Since they are basic names, I can set this labels in the constructor and if I need the +/- buttons in other place I can rename it. 

But the release callback I would prefer to set in the TimeGraph, following the usage of the button class.

Reply:
Refactored as I erased PlusButton and MinusButton class